app/main.py

```diff
@@ -2,0 +1,11 @@
+def copy_file(command: str) -> None:
+    split_command = command.split()
+    if split_command[1] == split_command[2]:
+        return
+
+    if split_command[0] == "cp":
+        with (
+            open(split_command[1], "r") as file_input,
+            open(split_command[2], "w") as file_output
+        ):
+            file_output.write(file_input.read())
```

Comment:
The function assumes that the command starts with 'cp', but does not handle the case when it does not. If the command does not start with 'cp', the function does nothing, which may be confusing for the user. Consider adding an error message or an exception in this case as well.